skbio/metadata/_interval.py

```diff
@@ -0,0 +1,699 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import operator
+from functools import wraps
```

Comment:
`import functools`, use `functools.wraps` in code below so it is clears where `wraps` comes from